lib/pacemaker/pcmk_output.c

```diff
@@ -66,68 +66,46 @@ colocations_header(pe_resource_t *rsc, rsc_colocation_t *cons,
     return retval;
 }
 
-PCMK__OUTPUT_ARGS("colocations-list", "pe_resource_t *", "gboolean", "gboolean")
-static int colocations_list(pcmk__output_t *out, va_list args) {
+PCMK__OUTPUT_ARGS("rscs-colocated-with-list", "pe_resource_t *", "gboolean")
+static int rscs_colocated_with_list(pcmk__output_t *out, va_list args) {
```

Comment:
Yep, it's the reverse :)

for "A colocated with B", the constraint is added to A->rsc_cons and B->rsc_cons_lhs

So looping through rsc->rsc_cons shows all constraints where rsc is colocated with some other resource, and looping through rsc->rsc_cons_lhs shows all constraints where some other resource is colocated with rsc.

Reply:
Despite all the staring at those functions that I did, I still got them backwards.  I think they're right now.